model/message.go

```diff
@@ -192,7 +192,7 @@ func MetricToRow(metric Metric, msg *InputMessage, dims []*ColumnWithType, idxSe
 			if strings.HasSuffix(dim.Name, "_topic") {
 				*row = append(*row, msg.Topic)
 			} else if strings.HasSuffix(dim.Name, "_partition") {
-				*row = append(*row, msg.Partition)
+				*row = append(*row, int32(msg.Partition))
```

Comment:
Can you please explain this change?

Reply:
Clickhouse driver return error for any type of column `driver.Batch.Append: clickhouse [AppendRow]: __kafka_partition clickhouse [AppendRow]: converting int to Int32 is unsupported`. I suggest use `Int32` type for this column in CH.